rtCommon/exampleInterface.py

```diff
@@ -0,0 +1,42 @@
+"""
+Examlpe remote interface, modify to add functionality. Remote interfaces can be easily
+created by subclassing the RemoteExtensible class.
+
+All methods in the subclass will be callable through an RPC interface
+which the classification script running in the cloud can access throug the client object.
+
+In addition to creating a subclassed RemoteExtensible, an instance of this object must also
+be added to the projectServerRPC.py file, such as exposed_bidsInterface. And a data service
+must run at the control room computer that has an instance of this class with dataRemote=False.
+
+The control room instance will be the actual instance and the projectServerRPC instance is
+a shell instance (dataRemote=True) that forwards request to the control room instance.
```

Comment:
Might be more consistent to switch 'shell' to 'stub', and removes any potential confusion about shell referring to something like `bash` or `zsh`